src/components/organisms/EntityDetails/EntityDetailsLayer.tsx

```diff
@@ -62,15 +62,21 @@ const EntityDetailsLayer: FC<PropsWithChildren<EntityDetailsLayerProps>> = ({
 
   const {data: rawExecutions, refetch} = useGetExecutions(
     {id, last: daysFilterValue},
-    {pollingInterval: PollingIntervals.long, skip: !isSystemAvailable || (!isClusterAvailable && Boolean(executions))}
+    {
+      pollingInterval: PollingIntervals.long,
+      skip: !isSystemAvailable || (storeDetails?.readOnly && Boolean(executions)),
```

Comment:
These should keep the previous conditions, and additionally have another for the expected case. Besides, it's important to take in account case:

* Agent is offline
* User reads the test from API
* Agent becomes online
* User should read the test from API again

With the current condition, it will skip update, as `storeDetails?.readOnly` will be `true`.